js/util/__tests__/extractProps.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, extractProps, PropExtractorConfig } from '../extractProps';
+import * as _ from 'lodash';
```

Comment:
Nit: separate third-party and first-party imports

Reply:
done